api/pkg/service/resource/resource.go

```diff
@@ -397,6 +398,7 @@ func versionInfoFromResource(r model.Resource) *resource.ResourceVersion {
 		ID:         r.ID,
 		Name:       r.Name,
 		Kind:       r.Kind,
+		HubURL:     fmt.Sprintf("%s/%s/%s/%s", r.Catalog.Name, r.Kind, r.Name, version),
```

Comment:
why "resource" obj's hub url has version?

Reply:
So this is for a `catalogKindNameVersion` api so for a specific version you get hubUrl as
`tekton/task/git-cli/0.1`